tests_integration/test_nativeapp.py

```diff
@@ -338,24 +338,28 @@ def test_nativeapp_version_create_and_drop(
             )
 
             # app package contains version v1
-            expect = row_from_snowflake_session(
-                snowflake_session.execute_string(
-                    f"show versions in application package {package_name}"
-                )
+            expect = snowflake_session.execute_string(
+                f"show versions in application package {package_name}"
             )
-            assert contains_row_with(expect, {"version": "V1"})
-            assert contains_row_with(expect, {"patch": 0})
+            actual = runner.invoke_with_connection_json(
+                ["app", "version", "list"], env=TEST_ENV
+            )
+            assert actual.json == row_from_snowflake_session(expect)
 
             result_drop = runner.invoke_with_connection_json(
                 ["app", "version", "drop", "v1", "--force"],
                 env=TEST_ENV,
             )
             assert result_drop.exit_code == 0
+            actual = runner.invoke_with_connection_json(
+                ["app", "version", "list"], env=TEST_ENV
+            )
+            assert len(actual.json) == 0
 
         finally:
             # make sure we always delete the package
             result = runner.invoke_with_connection_json(
-                ["app", "teardown", "--force"],
+                ["app", "teardown"],
```

Comment:
Why not --force? It seemed more prudent. Same with other tests.

Reply:
It is not necessary for these tests. 
1. The application is created by the CLI and therefore has a special comment.
2. The application package is created by the CLI (and therefore has a special comment) and its distribution is set to internal. 
In both the cases, the use of `--force` is not required.

Reply:
Sure, but that assumes that the test behaves as intended. We should not, in general, make this assumption. In the case where the test uncovers unexpected bahaviour (and, say, partial failure to create some objects), it'd be safer to use --force to make sure we clean up as much as we can.

Reply:
Agreed, we should definitely have `--force` here as this is just about cleanup, not making sure that certain preconditions are met. Can we re-introduce `--force` in a follow-up PR?

Reply:
#637